test/unit/cinstance_test.rb

```diff
@@ -712,7 +712,6 @@ class DescriptionValidationTest < ValidationsTest
       @cinstance.validate_human_edition!
 
       assert @cinstance.invalid?
-      assert @cinstance.errors[:description].presence
```

Comment:
In this case instead of just removing one assertion, the whole test needs to change, because now the behavior does not correspond to the name.

I suggest the following changes:

1. Add `not` ("require" => "not require") for 

```
test 'provider has multi apps enabled: require description if a human interaction is happening' do
test 'service requires intentions: require description if a human interaction is happening' do
```

2. Add the following for `DescriptionValidationTest` to ensure the validation does not fail due to missing `name`
```
    def setup
      super
      @cinstance.update(name: 'whatever')
    end
```

3. change all assertions within `DescriptionValidationTest` to 
```
assert @cinstance.valid?
```
Because now there is no description presence validation - in the conditions the tests check for.

Reply:
tests are failing need to fix all, I am working on it

Reply:
And `assert_nil @cinstance.errors[:description].presence`

Reply:
Actually if it is `valid?` then there are no errors. But it could assert
```
assert_nil @cinstance.description
```